coss/opensource_clubs/templates/opensource_clubs/entity_detail_page.html

```diff
@@ -1,109 +1,124 @@
+              <div class="mb-1 text-truncate">
+                <div class="icon d-inline-block mr-2">
+                  <img src="{% static 'img/icons/facebook-black-large.svg' %}" alt="web" class="img-fluid">
+                </div>
+                <img src="{% static 'img/icons/right-arrow-black.svg' %}" alt="web" class="pr-2">
+                <a href="{{ page.facebook_link }}" target="_blank">{{ page.facebook_link }}</a>
+              </div>
+            {% endif %}
+          </div>
+
+          {% with tags=page.tags.all %}
+            {% if tags.count %}
+              <div class="tags">
+                <div>
                   {% for tag in tags %}
-                    <a href="{% slugurl 'tags' %}?tag={{ tag }}">
-                      <button type="button">{{ tag }}</button>
+                    <a href="{% slugurl 'tags' %}?tag={{ tag }}" class="badge badge-primary">
+                      {{ tag }}
                     </a>
                   {% endfor %}
                 </div>
-              {% endif %}
-            {% endwith %}
-          </div>
-          <div class="col-md-8">
-            {{ page.description|richtext }}
-          </div>
-        </div>
-      </div>
-    </div>
-
-    <div class="row mt-5">
-      <div class="col-md-6">
-        <div class="profile-listing clearfix">
-          <div class="avatar">
-            <img src="{% static 'img/harvey-milk.png' %}" alt="fullname">
-          </div>
-          <div class="profile-details ml-3 mt-3">
-            Harvey Milk
-            <div class="profile-title">Club Captain</div>
-          </div>
+              </div>
+            {% endif %}
+          {% endwith %}
         </div>
-        <div class="profile-listing clearfix">
-          <div class="avatar">
-            <img src="{% static 'img/ansel-adams.jpg' %}" alt="fullname">
-          </div>
-          <div class="profile-details ml-3 mt-3">
-            Ansel Adams
-            <div class="profile-title">Regional Coordinator</div>
-          </div>
+        <div class="col-md-8 mt-3 mt-md-0">
+          {{ page.description|richtext }}
         </div>
       </div>
-      <div class="col-md-6 mt-3 mt-md-0">
-        {% include 'home/blocks/image_gallery.html' with gallery=page.gallery_images %}
-      </div>
     </div>
 
     <div class="row mt-5 mb-5">
       <div class="col-md-6">
-        <h4>
-          <img src="{% static 'img/icons/docs-black.svg' %}" alt="resources">
-          Club Resources
-        </h4>
-        <div class="row info-box">
-          <div class="col-md-12">
-            Add some club resources to help your club members get started and stay involved!
+        {% with members=page.members.all %}
+          {% if members.count %}
+            {% for member in members %}
+              <div class="profile-listing clearfix">
+                <div class="avatar">
+                  <img src="{% gravatar_url member.user.email 160 %}" alt="{{ member }}">
```

Comment:
This should be the `avatar_url` from the ClubProfile.

Reply:
If the `avatar` url is empty then we can fallback to the gravatar_url. Thoughts?